mods/ReadMe.txt

```diff
@@ -0,0 +1,63 @@
+||COPYRIGHT|| - ||Patent pending|| - ||CREATOR: Albin Karlsson|| - ||UNIVERSITY OF GOTHENBURG||
```

Comment:
Good job on the readme describing how to access the new map. I was able to follow it and it worked.